distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Telemetry.enso

```diff
@@ -0,0 +1,28 @@
+private
+
+import Standard.Base.Data.Dictionary.Dictionary
+import Standard.Base.Nothing.Nothing
+import Standard.Base.Data.Text.Text
+import Standard.Base.Data.Vector.Vector
+
+polyglot java import org.slf4j.LoggerFactory
+
+
+## Schedules a single telemetry message for sending.
+
+   Arguments:
+   - logger_name: Name of the (telemetry) logger
+   - msg: Message. Must be of format `<msg>: <args>`, where `<args>` has `<argname>=<argvalue>`
+          mapping, for example "my message: arg1=1, arg2=2".
+          See org.enso.logging.service.logback.telemetry.LogFormatter
+   - args: Arguments passed to the LoggingEvent object
+log logger_name:Text msg:Text args:Vector -> Nothing =
+    telemetry_logger = LoggerFactory.getLogger (_root_logger_name + "." + logger_name)
```

Comment:
Removal of `_root_logger_name` and simple:
```suggestion
    telemetry_logger = LoggerFactory.getLogger ("org.enso.telemetry." + logger_name)
```
would be way more straightforward.